### ISSUE TITLE
CorsairLink could not initialize or has no sensors.

### ISSUE THREAD

**Issue by Lithradus** [EvanMulawski/FanControl.CorsairLink#3]
Followed all the installation instructions on a fresh install of Fan Control. Closed iCUE and terminated all services in task manager before starting Fan Control.

usbdeview entry for my setup

CORSAIR iCUE Commander CORE	USB Composite Device	Unknown	Yes	Yes	No	No		630f0001801ccabae51f2af52091005f	2023-01-15 1:18:31 PM	2023-01-12 6:31:57 PM	1b1c	0c1c	1.00		00	00	00		JAMIE-PC			6&26eb1bce&0	usbccgp	@usb.inf,%GenericParent.SvcDesc%;Microsoft USB Generic Parent Driver	usbccgp.sys		(Standard USB Host Controller)		100 mA	2.00	USB Composite Device	10.0.25276.1000	Composite.Dev.NT	usb.inf	USB\VID_1B1C&PID_0C1C\630f0001801ccabae51f2af52091005f	Removable, UniqueID, SurpriseRemovalOK

**Comment by EvanMulawski** [comment]
Anything useful in `log.txt` in the FanControl directory? Also, make sure the Corsair Service service is still stopped - sometimes other Corsair services will keep starting it (on my PC, I changed the startup type to Manual).

I'm going to add more logging in the next beta release - hopefully that will help solve this issue. The CORE non-XT model is nearly impossible to get now.

**Comment by Lithradus** [comment]
This is all that's in log.txt:
2023-01-15 4:09:15 PM: CorsairLink could not initialize or has no sensors.

I got that after changing the services to manual and stopping them.

Unfortunately that is the commander that came with my cooler, just my luck I've got an oddball device.
If there is anything I can do to help test please don't hesitate.

**Comment by EvanMulawski** [comment]
When you get a chance, can you post any other devices in USBDeview that have `1b1c` as the Vendor ID?

**Comment by Lithradus** [comment]
0000.0014.0000.010.000.000.000.000.000	USB Input Device	HID (Human Interface Device)	Yes	Yes	No	No			2023-01-15 3:52:12 PM	2023-01-15 3:52:12 PM	1b1c	0c1c	1.00		03	00	00		JAMIE-PC			7&20b69190&0	HidUsb	@input.inf,%HID.SvcDesc%;Microsoft HID Class Driver	hidusb.sys		(Standard system devices)				USB Input Device	10.0.25276.1000	HID_Inst.NT	input.inf	USB\VID_1B1C&PID_0C1C&MI_00\6&26eb1bce&0&0000	SurpriseRemovalOK					


0000.0014.0000.010.000.000.000.000.000	USB Input Device	HID (Human Interface Device)	Yes	Yes	No	No			2023-01-15 3:52:12 PM	2023-01-15 3:52:12 PM	1b1c	0c1c	1.00		03	00	00		JAMIE-PC			7&387a1dd5&0	HidUsb	@input.inf,%HID.SvcDesc%;Microsoft HID Class Driver	hidusb.sys		(Standard system devices)	Corsair RGB Commander Core (CRCC) #1 (USB)			USB Input Device	10.0.25276.1000	HID_Inst.NT	input.inf	USB\VID_1B1C&PID_0C1C&MI_01\6&26eb1bce&0&0001	SurpriseRemovalOK

**Comment by EvanMulawski** [comment]
Closing in favor of #2 to keep related communication in one place since it's the same problem. Please subscribe to that issue.
